tensorflow/core/kernels/BUILD

```diff
@@ -4079,6 +4087,7 @@ filegroup(
         "fake_quant_ops.cc",
         "fifo_queue.cc",
         "fused_batch_norm_op.cc",
+        "whole_file_read_ops.cc",
```

Comment:
I was just thinking the whole_file_read_ops.cc, but if it's related to this PR, it's fine to keep it.